Cesium.Runtime/StdIoFunctions.cs

```diff
@@ -23,6 +52,39 @@ public static void PutS(byte* str)
             // return EOF; // TODO[#156]: Uncomment
         }
     }
+    public static int PutChar(byte character)
+    {
+        try
+        {
+            Console.Write((char)character);
+            return 0;
```

Comment:
According to the standard:
> The `putchar` function returns the character written. If a write error occurs, the error indicator for
> the stream is set and `putchar` returns `EOF`.

```suggestion
            return character;
```